docs/modules/ROOT/pages/index.adoc

```diff
@@ -910,14 +910,13 @@ An attacker could make use of visually-similar Unicode characters to mislead an
 
 NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/117[issue #117]): Discuss and add guidance.
 
-==== Validation of credential status
+==== Revocation of credential
 
-* Does the _<<_identity_assertion_consumer,identity assertion consumer>>_ need to do real-time checks to ensure that the _<<_named_actor,named actor’s>>_ credential is still considered valid? What harms or risks may accrue to the _<<_identity_assertion_consumer,identity assertion consumer>>_ in the process of making online inquiries about such status?
-* If a bad actor makes it into the system, then how does that ID get blocked? Is there an OCSP, CRL, bad actor database, etc.? If so, what is the governance for such a list and who maintains it?
-* What if the bad actor is in an ingredient and not the primary piece of content?
-* What is displayed to the end-user in both the primary content and ingredient scenarios?
+A _<<_credential_holder,credential holder>>_ such as credential compromise or acting in bad faith that would lead to the credential being revoked.
```

Comment:
Note that credentials may be revoked as part of the lifecycle of the identity and of the credential. 
For example, an individual who changes part of their identity record -- their name or address -- will likely have their old credential revoked and a new one issued to them, so that the credential reflects the current state of the source identity record. Also: some credentials may have expiry dates enforced by revocation.
Two implications of this:
- Revocation is a poor signal that credential's subject did anything wrong
- Revocation will be a common (and largely inevitable) event
Possibly the key point is that the credential was not revoked at the time the identity assertion was signed